deeppavlov/dataset_readers/squad_dataset_reader.py

```diff
@@ -51,6 +57,8 @@ def read(self, dir_path: str, dataset: str = 'SQuAD', *args, **kwargs) -> Dict[s
             self.url = self.url_squad
         elif dataset == 'SberSQuAD':
             self.url = self.url_sber_squad
+        elif dataset == 'MultiSQuAD':
+            self.url = self.url_multi_squad
         else:
             raise RuntimeError('Dataset {} is unknown'.format(dataset))
```

Comment:
It would be great also to have this documented in class docstring.

Reply:
It is documented here: https://github.com/deepmipt/DeepPavlov/blob/feat/squad_with_no_ans_token/deeppavlov/dataset_readers/squad_dataset_reader.py#L38

Reply:
I mean documenting exceptions, that could be raised...

Reply:
fixed